flags.go

```diff
@@ -50,10 +50,15 @@ var (
 		Name:  "tlsverify",
 		Usage: "Use TLS and verify the remote",
 	}
+	flOverCommit = cli.IntFlag{
+		Name:  "overcommit, oc",
+		Usage: "OverCommit to apply on resources",
+		Value: 105,
```

Comment:
@tnachen part of this change stems from [this comment](https://github.com/docker/swarm/pull/227#issuecomment-69090266) in #227. The concept is based off linux's memory overcommit.

This bit of the change was actually made to allow undercommit. There are cases where you might not want a container to get allocated to a box where the container will consume almost all of the box's resources. In this case you would set overcommit to a value less than `100`.